Frontend/android-native/app/src/main/java/in/juspay/mobility/utils/NotificationUtils.java

```diff
@@ -189,11 +184,36 @@ public static Notification createNotification (Context context, String title, St
 
     public static void showAllocationNotification (Context context, String title, String msg, JSONObject data, String imageUrl, JSONObject entity_payload){
         try{
+
+            final SimpleDateFormat f = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.SSS'Z'",new Locale("en","US"));
+            f.setTimeZone(TimeZone.getTimeZone("IST"));
+            String currTime = f.format(new Date());
             String notificationType = new String(data.getString("notification_type"));
             String channelId;
-            if (ALLOCATION_TYPE.equals(notificationType)) {
-                System.out.println("showNotification:- "+ notificationType);
+            SharedPreferences sharedRemovePref = context.getSharedPreferences(
```

Comment:
@KiranBKL you can use static also since its independent of activity, you can revert your changes and squash